components/TripList/index.js

```diff
@@ -0,0 +1,22 @@
+import useSWR from "swr";
+import Trip from "../Trip";
+
+const fetcher = (...args) => fetch(...args).then((res) => res.json());
+
+export default function TripList() {
+  const { data: trips, isLoading } = useSWR(`/api/trips`, fetcher);
+  console.log(trips);
+  if (!trips || isLoading) {
+    return <>is Loading...</>;
+  }
+
+  const { _id } = trips;
+
+  return (
+    <ul>
+      {trips.map((trip) => (
+        <Trip key={_id} {...trip} />
```

Comment:
Please see my comment above: 
```suggestion
        <Trip key={trip._id} {...trip} />
```
Additionally. I'm not a big fan of spreading the props. It might be a bit more effective, but it has the disadvantage that it's not exactly clear which properties are passed to the component.  But this is up to you 🙃